### DISCUSSION TITLE
Week 1 - Feedback from an event with 90 people expected

### DISCUSSION THREAD

**Discussion by Dayk0** [mozilla/hubs#3868] [Hubs Cloud Questions] [answered]
Hello, feedback from yesterday with an event which was to welcome 60-70 participants in a room. I was with the c5.24xlarge enterprise maximum server for added security. The server was working fine but we made the mistake of adding ten videos in space, which caused big technical problems for the users to the point that it was impossible to speak even with 30 people in the room because the audio stream scorched sharply. In addition, we had spatialized the sound so that everyone could hear.
The scene was already very well optimized with around 15mb, very few polygons, etc. but the videos were really the biggest problem. They were in automatic reading.

I am doing the same event again next week but we decided to remove all the videos and have PDFs / images there instead.
I would like to know if this time, in your opinion, without the videos, with pdf / images in replacement and an even lighter scene (10mb), I could not have a problem with the audio ? There will be around 160-180 people spread over 2 different rooms. The sound will be spatialized and maybe half of the people will talk to each other while the others watch the pdf.
Also, can you tell me why the mics started to roast? Is the video audio stream server linked with the microphones audio stream? Or the sound starts to roar because the computers are not powerful enough because of a room too loaded with video?

Thank you, see you soon.

**Comment by robinkwilson** [comment, marked as answer]
Hello!

Oh boy! Yes we suggest limiting only 3 videos to a room for performance reasons. 

In addtion for the same performance reasons, the most people that can be avatars in the room should be limited to 24 people at once. You can have 75 people **in the lobby** (not avatars) listening and flying around if you need that many people connecting to one room. 

I really recommend splitting up the number of people over a lot more rooms to 20 or so people per room.

We really appreciate the feedback!! Thanks for the write up! Hope this helps.

**Reply by robinkwilson** [reply]
Also wanted to mention, where the bottleneck is since you had a beefy c5.24xlarge server.

Was this an enterprise multi-server instance or just 1 server enterprise?

The bottleneck here is definitely the connected clients. All that video + audio data from every client drives up difficulty for a smooth performance.

**Reply by Dayk0** [reply]
Thanks for your feedback. I will do another test this afternoon, as expected, with a very very light scene and a few PDFs, I deleted the videos and turned off the ability to put objects. We will see what happens.
It was a multi-server instance with 2 apps and 2 streaming for added security. I think that indeed it came from the connected clients.

**Reply by yengalvez** [reply]
Hi Robin, I'm using Hubs Cloud. Is there someway to show on a landing page the amount of users in a room? Then I can help people know here they can join with less users. Thanks.
